### DISCUSSION TITLE
Time based numerical tracker

### DISCUSSION THREAD

**Discussion by Antimatter543** [iSoron/uhabits#1047] [Feature Requests] [answered]
So far (to my knowledge), only base 10 numerical systems are used in the latest version of uhabits (e.g 3 miles ran, 50 pages read, etc.) but there is no good way to track time spent doing something (e.g 30 minutes study). 

The best option available so far is trying to convert hours -> base 10, so that 0.5 'hours' is a thing but then it gets tricky when you reach 0.90,  0.95, ... (Most people tend not to be readily able to convert 0.85 hours into a meaningful metric).

**Comment by hiqua** [comment, marked as answer]
https://github.com/iSoron/uhabits/discussions/953
